src/workflow/ArcanistLandWorkflow.php

```diff
@@ -1523,115 +1523,6 @@ private function restoreBranch() {
   }
 
 
-  /**
-   * Check if a diff has a running or failed buildable, and prompt the user
-   * before landing if it does.
-   */
-  private function checkForBuildables($diff_phid) {
-    // Reset ongoing builds value.
-    $this->uberOngoingBuildsExist = false;
-
-    // Try to use the more modern check which respects the "Warn on Land"
-    // behavioral flag on build plans if we can. This newer check won't work
-    // unless the server is running code from March 2019 or newer since the
-    // API methods we need won't exist yet. We'll fall back to the older check
-    // if this one doesn't work out.
-    try {
-      $this->checkForBuildablesWithPlanBehaviors($diff_phid);
-      return;
-    } catch (ArcanistUserAbortException $abort_ex) {
-      throw $abort_ex;
-    } catch (Exception $ex) {
-      // Continue with the older approach, below.
```

Comment:
None of the code below here was actually being run, so I removed it.

I also renamed `checkForBuildables` to `ensureBuildableIsPassing` which is clearer about what the function is doing.